### PR TITLE
Fix/add images to datasession

### PR DESCRIPTION
**Pull request by capetillo**
### Description
**Background:**
There was a bug where we couldn't add images to existing data sessions because the structure of the keys from images objects changed

**Implementation:**
Fairly simple implementation. In `addImagesToExistingSession` found in Project View, I changed the `source` and `basename` under `inputData` to `image.url` and `image.basename`, respectively, because we were sending empty objects. Now we can add images to existing data sessions : )

### CODE REVIEW

File: src/components/DataSession.vue [hunk 1]
```diff
@@ -39,31 +50,46 @@ onMounted(() => {
...
-          :operations="data.operations"
-          @add-operation="addOperation"
-        />
+        <v-img
+          :src="image.source"
+          :alt="image.basename"
+          cover
+          aspect-ratio="1"
+        >
+          <!-- <template #placeholder>
```
Comment (mgdaily) [inline src/components/DataSession.vue:68]: Commented code needed?

File: src/components/DataSession.vue [hunk 2]
```diff
@@ -28,9 +28,20 @@ async function addOperation(operationDefinition) {
 	await fetchApiCall({url: url, method: 'POST', body: operationDefinition, successCallback: emit('reloadSession'), failCallback: handleError})
 }
 
+const displayImages = (data) => {
+	images.value = data.input_data
+	console.log('images.value', images.value)
```
Comment (mgdaily) [inline src/components/DataSession.vue:33]: Extraneous console log? I think having this inline like it was before is just fine if it's a one-liner

Comment (capetillo) [reply src/components/DataSession.vue:33]: oopsi

File: src/components/DataSession.vue (hunk 1 above)
Comment (capetillo) [reply src/components/DataSession.vue]: deleted!

**Comment by mgdaily** [comment]
Looks like there are some merge conflicts that need to be resolved!

**Review by mgdaily — APPROVED** [review]
lookit that small diff! nice work

**Pull request merged**